app/assets/javascripts/components/data_portal/modals/modal-save-widget.js

```diff
@@ -43,28 +44,116 @@
       }];
     },
 
-    _setPosition: function () {
+    //  *** localStorage private methods ***
+
+    /**
+     * @return array - Returns an array of indicators or an empty array
```

Comment:
It's a good thing you've started to add comments, but you should check the [proper syntax](http://usejsdoc.org/about-getting-started.html).

After `@return` or `@param`, you must write the type of the variable between braces and then the name of the param in the case of `@param`.

Note that `@params` doesn't exist, it's always `@param`.